dnf-docker-test/README.md

```diff
@@ -37,6 +37,10 @@ To rebuild the Docker image you can use the following command:
 $ cd dnf_docker_test/
 $ sudo docker build -t jmracek/dnftest:1.0.2 .
 ```
+To rebuild test-1 repository go into test-1 directory with rpm files and comps-f23.xml file and use following command:
+```
+$ createrepo -g comps-f23.xml .
```

Comment:
I have a plan to change repo creator when tests for group upgrade will be introduced.